jest-preprocessor.js

```diff
@@ -1,12 +1,18 @@
-var coffee = require('coffee-script');
+const babelJest = require('babel-jest');
 
 module.exports = {
-  process: function(src, path) {
+  canInstrument: babelJest.canInstrument,
+  getCacheKey: babelJest.getCacheKey,
+  process: function(src, path, ...rest) {
     if (path.match(/\.(s?css)/)) {
       return;
     }
-    if (coffee.helpers.isCoffee(path)) {
-      return coffee.compile(src, {'bare': true});
+    if (
+      path.indexOf('node_modules') === -1 ||
+      path.indexOf('bluecore-ui-kit') > -1 ||
+      path.indexOf('bluecore-api-lib') > -1
```

Comment:
I think `bluecore-ui-kit` and `bluecore-api-lib` is unnecessary here.

Reply:
I think you're right 😄